elearning-frontend/src/components/ProfileSettings.js

```diff
@@ -80,7 +79,7 @@ const ProfileSettings = (props) => {
         }).then(response => {
             Toastify("Succesfully saved user information")
         }).catch(error => {
-            Toastify(Object.values(error.response.data.errors)[0][0])
+            Toastify(!((typeof error.response.data.errors) === 'undefined') ? Object.values(error.response.data.errors)[0][0]  : error.message)
```

Comment:
Must apply DRY. Dont Repeat Yourself

Maybe you can create  an error handler function that will build error message instead doing this ternary over and over again. Apply to other error handler also.